app/views/posts/index.html.erb

```diff
@@ -1,10 +1,13 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
-  </head>
+<section class="user-container d-flex-col">
+  <%= render partial: 'users/user_card', object: @user, as: :user %>
 
-  <body>
-    <h2>List of posts, general views</h2>
-  </body>
-</html>
\ No newline at end of file
+  <%= render partial: 'posts_comments', collection: @posts, as: :post %>
```

Comment:
- Please use the method `five_recent_comments` to render the recent comments for each post as required.

Reply:
Hi @afanimartin, if you go to the `app/views/posts/_posts_comments`, I'm using a method named `comments_exist` that is in the `helpers/application_helper` where I'm using the method to render the recent comments.